src/main.py

```diff
@@ -4,11 +4,16 @@
 from src.models.predict_model import predict_model
 from src.visualization.visualize import plots
 import logging
+from codecarbon import EmissionsTracker
 
 logging.basicConfig(level=logging.WARN)
 logger = logging.getLogger(__name__)
 
 def ETL():
+    # We will track CO2 emissions for each step of our pipeline
+    tracker = EmissionsTracker()
+    tracker.start()
```

Comment:
We want to track the emissions on every single function. I suggest to create different trackers, one for the loading, another for the training...